static/templates/chat/spell-card.html

```diff
@@ -1,4 +1,12 @@
-<div class="pf2e chat-card item-card" {{#if data.item}}data-embedded-item="{{data.item}}"{{/if}} data-spell-lvl="{{data.spellLvl}}" data-actor-id="{{actor.id}}" data-item-id="{{item.id}}" {{#if tokenId}}data-token-id="{{tokenId}}"{{/if}}>
+<div class="pf2e chat-card item-card"
+    {{#if data.item}}data-embedded-item="{{data.item}}"{{/if}}
+    data-spell-lvl="{{data.slotLevel}}"
```

Comment:
```suggestion
    data-spell-lvl="{{data.castLevel}}"
```
If this is sticking around for backwards compatibility, for the spellcard this was always the cast level.

Reply:
But there was no cast level sent to the template

Reply:
![image](https://user-images.githubusercontent.com/1286721/181644303-c58109f3-1e8d-4b36-b82c-1765df0e5500.png)

spell level is set to 4 for a cantrip. That means its the cast level, not the slot level.

Reply:
In general, what spell-lvl refers to in live changed depending on what phase you were in. In the actor sheet it was slot level, but in the chat card it was cast level.

Reply:
Ah, because I forgot to clarify, this is why I said it was "just cantrips" and that an alternative is to adjust drag/drop code. Spell level being "what it will be casted as" vs "what it got casted as" wasn't really broken as a paradigm until you got to cantrips. An alternative was to alter the sibling checks to prioritize cantrips over spell level, and make spell level the "will cast at" level always.

Having slot level and cast level separate might be easier to wrap someone's head around though.